sass/directives/_flyout.scss

```diff
@@ -87,3 +87,122 @@
     }
   }
 }
+
+// Use the mobile-subnav mixin for the subnav bars that appear on our subscription / product pages
+// Displays a single link with a flyout
+@mixin mobile-subnav-flyout() {
```

Comment:
As @MKwenhua mentioned, you don't need the `()` for mixins that lack params. It matches what we've been doing stylistically, and is one of the things that makes it ever-so-slightly easier to sight-read mixins.